ssas/service/admin/router.go

```diff
@@ -34,7 +34,7 @@ func Server() *service.Server {
 
 func routes() *chi.Mux {
 	r := chi.NewRouter()
-	r.Use(service.NewAPILogger(), service.ConnectionClose)
+	r.Use(service.NewAPILogger(), service.ConnectionClose, requireBasicAuth)
```

Comment:
Do any of the tests show that the request will fail without the basic auth header?